README.md

```diff
@@ -0,0 +1,85 @@
+# [Willow image library]((https://pypi.org/project/wagtailmedia/)
```

Comment:
Markdown rendering is broken

```suggestion
# [Willow image library](https://pypi.org/project/wagtailmedia/)
```

Reply:
sorted. I copied this and the badges from wagtailmedia and the above URL was wrong 🙈 😂